Sources/FluentBenchmark/Tests/SoftDeleteTests.swift

```diff
@@ -1,9 +1,10 @@
 extension FluentBenchmarker {
     public func testSoftDelete() throws {
-        try self.testSoftDelete_model()
-        try self.testSoftDelete_query()
-        try self.testSoftDelete_onBulkDelete()
-        try self.testSoftDelete_forceOnQuery()
+//        try self.testSoftDelete_model()
+//        try self.testSoftDelete_query()
+//        try self.testSoftDelete_onBulkDelete()
+//        try self.testSoftDelete_forceOnQuery()
+        try self.testSoftDelete_parent()
```

Comment:
did you intentionally leave these commented out?

Reply:
Oops, no. Thank you!